lib/tesla_api/client.rb

```diff
@@ -21,13 +21,23 @@ def initialize(
       @access_token_expires_at = access_token_expires_at
       @refresh_token = refresh_token
 
+      retry_options = {
+        max: 3,
+        interval: 0.5,
+        interval_randomness: 0.5,
+        backoff_factor: 2,
+        methods: %i[get post],
```

Comment:
I'm not sure we want to retry on POST requests. The failure may be within Tesla's infrastructure on the return path, so after the request was made to the car. Some operations are not idempotent and will return differently upon retry.

Reply:
Absolutely a valid point! I also had that in mind, but mostly focused on my use cases where the worst case would be triggering a command twice (e.g. unlocking doors).

Maybe it's not a good idea to have that as a default.